### PR TITLE
TP UI follow-up

### PR DESCRIPTION
**Pull request by garvankeeley**
@bryanbell sat with me and we went through a number of UI issues that are outstanding with the TP changes. 
Additionally, strings were updated.
Some changes have an effect on the entire app, for instance our min font size of 11 used for small headings on menus, is now 12 point font.

If needed, I can split this into separate PRs.

### CODE REVIEW

File: Client/Frontend/Widgets/PhotonActionSheet/TrackingProtectionMenu.swift [hunk 1]
```diff
@@ -139,37 +140,47 @@ extension PhotonActionSheetProtocol {
...
             self.showDomainTable(title: title, description: desc, blocker: blocker, categories: [BlocklistCategory.fingerprinting])
         }
-        let cryptomining = PhotonActionSheetItem(title: Strings.TPCryptominersBlocked, iconString: "tp-cryptominer", accessory: .Disclosure) { action in
+        let cryptomining = PhotonActionSheetItem(title: Strings.TPCryptominersBlocked, iconString: "tp-cryptominer", accessory: .Disclosure) { action, _ in
             let title = String.localizedStringWithFormat(Strings.TPListTitle, action.title)
             let desc = Strings.TPCategoryDescriptionCryptominers
             self.showDomainTable(title: title, description: desc, blocker: blocker, categories: [BlocklistCategory.cryptomining])
         }
 
-        let addToWhitelist = PhotonActionSheetItem(title: Strings.TPDisableTitle, iconString: "menu-TrackingProtection-Off") { _ in
```
Comment (isabelrios) [inline Client/Frontend/Widgets/PhotonActionSheet/TrackingProtectionMenu.swift]: @garvankeeley I'm not sure if this is the reason, but the identifier `menu-TrackingProtection-Off`to enable/disable TP is not working anymore (I run the tests on this branch on BB) it is found, but tapping on it does not enable/disable the switch. Could you please keep it? or add an ID for the switch button and make it visible (now I don't see it printed as an element 🙈 )

**Comment by isabelrios** [comment]
@garvankeeley sorry for jumping in here...I'm trying to keep the tests working or at least have them failing the less time possible...is it correct that after these changes if you disable ETP from settings menu, the shield icon is not displayed anymore (in same site or different with/without blockers)? Thanks!

**Comment by garvankeeley** [comment]
Thanks for looking at this @isabelrios ! Yes, the shield disappears now when ETP is disabled in settings. Also note, the shield has moved to the left of the lock icon, although that shouldn't affect the tests.

File: Client/Frontend/Widgets/PhotonActionSheet/TrackingProtectionMenu.swift (hunk 1 above)
Comment (garvankeeley) [reply Client/Frontend/Widgets/PhotonActionSheet/TrackingProtectionMenu.swift]: Let me look at fixing this...

**Pull request merged**